src/main/java/seedu/mtracker/model/Instrument.java

```diff
@@ -32,13 +48,15 @@ public String getSentiment() {
         return sentiment;
     }
 
-    public abstract String toString();
+    public String toString() {
+        return TextUi.createBoxDisplay(getStatusIcon());
+    }
 
     public abstract String getType();
 
     public String toList() {
-        return TYPE_HEADER + getType() + System.lineSeparator()
-                + NAME_HEADER + getName() + System.lineSeparator()
+        return TYPE_HEADER + TextUi.createBoxDisplay(getStatusIcon()) + getType()

```

Comment:
I think getType should appear first so its 
Type: stock [x], not Type: [x] stock

Reply:
The style of this, feel free to modify if u feel it would look better